c-ares-config.cmake.in

```diff
@@ -7,11 +7,11 @@ include("${CMAKE_CURRENT_LIST_DIR}/c-ares-targets.cmake")
 set(c-ares_LIBRARY c-ares::cares)
 
 if(@CARES_SHARED@)
-	add_library(c-ares::cares_shared INTERFACE IMPORTED)
+	add_library(c-ares::cares_shared SHARED IMPORTED)
 	set_target_properties(c-ares::cares_shared PROPERTIES INTERFACE_LINK_LIBRARIES "c-ares::cares")
 	set(c-ares_SHARED_LIBRARY c-ares::cares_shared)
 elseif(@CARES_STATIC@)
-	add_library(c-ares::cares_static INTERFACE IMPORTED)
+	add_library(c-ares::cares_static SHARED IMPORTED)
```

Comment:
I doubt if I did the correct thing for the "CARES_STATIC" case: should I have used the STATIC target here?

Reply:
That won't work that way. Why do you have to use such an ancient CMake version in the first place?

Reply:
Hi Gregor, 

Thanks for replying to my RFC.

Unfortunately in my build environment currently CMake 2.8.12 is a given.
The CmakeLists.txt file claims it is supported:

`CMAKE_MINIMUM_REQUIRED (VERSION 2.8.12)`

What do you mean with "that won't work that way"?
Did I break it for the static build? For my build CARES_SHARED is enabled, so maybe only the change on line 10 makes sense? Or is there another way to keep 2.8.12 supported?